### PR TITLE
cmd/operator: fix periodFullGC only exec once

### PR DESCRIPTION
**Pull request by wilhelmguo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 ### Fixed
 
+- Fix cmd/operator: periodFullGC only exec once.
```
Comment (hongchaodeng) [inline CHANGELOG.md]: Fix periodFullGC only executed once problem.

**Comment by hongchaodeng** [comment]
LGTM after minor nits

File: CHANGELOG.md (hunk 1 above)
Comment (hongchaodeng) [inline CHANGELOG.md]: @wilhelmguo 
Can you change the above line to my comment?

Comment (wilhelmguo) [inline CHANGELOG.md]: @hongchaodeng OK

**Comment by wilhelmguo** [comment]
@hongchaodeng I have change the CHANGELOG. Please check it.

**Comment by xiang90** [comment]
lgtm

**Comment by hongchaodeng** [comment]
@wilhelmguo Thanks!

**Pull request merged**